style.css

```diff
@@ -1,438 +1,974 @@
+    margin-bottom: 45px;
+    margin-left: 130px;
+  }
+
+  .social-form {
+    display: flex;
+    list-style-type: none;
+    color: #505f79;
+    margin-left: 60px;
+  }
+
+  .social-form > li {
+    margin: 10px;
+  }
+
+  .footer-line-desktop {
+    display: none;
+  }
+}
+
+/* Desctop */
+
+@media only screen and (min-width: 768px) {
+  .nav-container {
+    background-color: #3a3c39;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .burger {
+    display: none;
+  }
+
+  .name {
+    margin-left: 9.625rem;
+  }
+
+  .shortcut-contener {
+    margin-right: 12.875rem;
+    align-self: center;
+  }
+
+  .shortcut-mail {
+    margin-bottom: -0.5625rem;
+  }
+
+  .shortcut {
+    margin-right: 2rem;
+    color: white;
+    text-decoration: none;
+  }
+
+  .home-container {
+    background-color: #262626;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: start;
+    background-image: url("image/biggroup.png");
+    background-position: right 0% top 0%;
+    background-repeat: no-repeat;
+    background-size: 31.25rem;
+    height: 49.875rem;
+  }
+
+  .home-container-item {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .greeting {
+    color: #ff6b00;
+    font-family: “Crete Round”, sans-serif;
+    position: static;
+    width: 566px;
+    height: 160px;
+    left: 0;
+    top: 12rem;
+    font-weight: 400;
+    font-size: 4.5rem;
+    line-height: 5.125rem;
+    margin-left: 8.9375rem;
+  }
+
+  .job {
+    font-size: 2rem;
+    font-weight: 700;
+    color: #fff5e1;
+    width: 35.375rem;
+    height: 1.5rem;
+    top: 23.5rem;
+    margin-left: 8.9375rem;
+    font-family: "Inter", sans-serif;
+  }
+
+  .about-me {
+    position: static;
+    width: 35.375rem;
+    height: 7.6875rem;
+    left: 0;
+    top: 26.5rem;
+    font-family: "Inter", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.25rem;
+    line-height: 2rem;
+    color: #f4f5f7;
+    margin-left: 8.9375rem;
+  }
+
+  .social-link-list {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    list-style-type: none;
+  }
+
+  .social-link-item {
+    margin-bottom: 10px;
+  }
+
+  .react-work {
+    display: none;
+  }
+
+  .react-work-desktop {
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    margin: 0.8125rem 8.9375rem 8.125rem;
+    height: 90rem;
+    background-image: url("image/react-desktop.png");
+    background-repeat: no-repeat;
+    background-size: 52% 28rem;
+    background-position: 0% 13%;
+  }
+
+  .home-title {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+  }
+
+  .text-title {
+    position: static;
+    width: 19.6875rem;
+    height: 3.25rem;
+    left: 0;
+    top: 0;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    align-items: center;
+    color: #091e42;
+  }
```

Comment:
Please kindly fix the margin and padding on the recent works title and please lets stick to the design on Figma.

This is your design:
![Screen Shot 2022-04-29 at 4 50 59 PM](https://user-images.githubusercontent.com/79506105/165970367-dd43dc00-69eb-40cd-bf5d-166ccaa787ae.png)

Figma design:


![Screen Shot 2022-04-29 at 4 51 30 PM](https://user-images.githubusercontent.com/79506105/165970584-5cad6241-c882-4f31-bb25-31123f9c8ba6.png)

Your recent works title should be exactly as it is in the figm template please let work on that and carefully inspect the template

Reply:
Hi @KayLemba 
Thanks for the review.
But I don't understand some of your comments about "Please add a screenshot showcasing your UI and a project description giving".

Reply:
Add a screen shot of your current webpage which is the desktop version add it to your Readme file look at the link I shared with you on the readme template

Reply:
Then a project description is details about the project for example you can say: "This is a desktop version portfolio.... etc write about the project you are building look at the link about the Readme file I shared with u

Reply:
Hi @KayLemba 
I think it is not on the requirement please check it out, Sir.